quality-assurance/test-strategy.md

```diff
@@ -4,11 +4,9 @@ This strategy details how we add value through testing at the UKHO. Our teams ad
 
 Our core principles are:
 
-* [Test at every stage of the development process](./testing-principles.md#we-test-at-every-stage-of-the-development-process)
+* A Shift Left test approach should be adopted.
```

Comment:
Not everyone reading this document is necessarily going to know what "Shift Left" is off the tops of their heads.
How about rephrasing this as `Test early and often using Shift Left methodology` and link it to a site on shift left testing?